2017-10/agenda.md

```diff
@@ -4,40 +4,31 @@
 
 ## 2017-10-06 Day One
 
-|    Time Slot    | Room |      Session     |        Description       |
-| --------------- | ---- | ---------------- | ------------------------ |
-|  1:30p - 3:00p  | TBD  | Core Plenary     | All-Collaborator Plenary |
-|  3:00p - 4:30p  | TBD  | Working Sessions | Working Sessions         |
-|  4:30p - 5:00p  | TBD  | Day 1 Summaries  | Summary of Day 1 Results |
+|    Time Slot    | Room |      Session           |        Description       |  Location                       |
+| --------------- | ---- | ----------------       | ------------------------ |-------------------------------- |
+|  1:30p - 3:00p  | TBD  | Core Plenary           | All-Collaborator Plenary | West Ballroom A                 |
+|  3:00p - 4:00p  | TBD  | Core Values--all hands | All hands discussion     | West Ballroom A                 |
+|  4:00p - 5:00p  | TBD  | Working Sessions       | Working Sessions         | Select breakout rooms from map  |
+|  4:30p - 5:00p  | TBD  | Day 1 Summaries        | Summary of Day 1 Results | West Ballroom A                 |
```

Comment:
I imagine the 4:00p - 5:00p session is not supposed to overlap with the 4:30p - 5:00p session?